src/RPC/RPC.php

```diff
@@ -32,9 +32,15 @@ class RPC implements RPCInterface
 
     /**
      * @var positive-int
+     * @deprecated since v3.2.1.
      */
     private static int $seq = 1;
```

Comment:
we can make it non-static to have a personal state for each RPC object

Reply:
My design idea is that sequence count should be related to Reply Object, not RPC objects

Reply:
This property is retained solely for backward compatibility and will be used only if the user uses a custom implementation of RelayInterface without the required method.

Reply:
> @shellphy BTW, what was the reason for moving the counter to Relay? If you could create multiple instances of RPC and create separate instances of Relay in them? In that case, it’s enough to just make the **$sec** property **non-static**.

I think about the relay connection pool, where there may be multiple rpc associated with the same relay, and it might make more sense for `$seq` to be associated with the relay. 

Of course, in general, the connection pool will also use locks or other mechanisms to ensure that a relay will not be associated with multiple RPCS at the same time, so it is possible to associate seq with rpc, and it is simpler. 

Never mind, it is still related to rpc, maybe I think the problem is complicated.